daemon/container.go

```diff
@@ -677,8 +677,10 @@ func (container *Container) Kill() error {
 	}
 
 	// 1. Send SIGKILL
-	if err := container.KillSig(9); err != nil {
+	if err := container.KillSig(9); err != nil && err != syscall.ESRCH {
```

Comment:
This can be rewritten like
```
if err := container.KillSig(9); err != nil {
    if err != syscall.ESRCH {
        return err
    }
    log.Debugf("Cannot kill process (pid=%d): no such process.", container.GetPid())
}
```
which is more readable IMO